lib/resources/dstu2/medication_statement.yaml

```diff
@@ -52,7 +48,6 @@ fields:
 
 - name: reasonForUse[x]
   required: 'no'
```

Comment:
capitalize any No's that aren't already

Reply:
[Fixed that, no cap](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-5321e3dd849f4c21936b5a38c0bf2092L50)